packages/perseus/src/widgets/interactive-graphs/graphs/utils.ts

```diff
@@ -237,3 +239,103 @@ export function getQuadraticXIntercepts(
 
     return [x1, x2];
 }
+
+export function getAngleFromPoints(points: Coord[], i: number) {
```

Comment:
Love this! It might be good to add a comment or two just to help save future devs some time.